client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -454,6 +458,46 @@ export class List extends React.Component {
 		);
 	}
 
+	renderPrimaryDomain() {
+		const { domains, selectedSite, translate } = this.props;
+		const primaryDomain = find( domains, 'isPrimary' );
+
+		if ( this.isLoading() || ! primaryDomain ) {
+			return <ListItemPlaceholder />;
+		}
+
+		return [
+			<CompactCard className="list__header-primary-domain" key="primary-domain-header">
+				<div className="list__header-primary-domain-info">
+					{ translate( 'Primary domain' ) }
+					<InfoPopover iconSize={ 18 }>
```

Comment:
Can you somehow vertically align this icon to the text as it is in the figma file and add 5px of left margin?